apps/frontend/src/modules/Currency/services/currencyService.ts

```diff
@@ -0,0 +1,15 @@
+import { getSubstrateAssetBalances } from '@/modules/TFChain/services/tfchainService';
+
+export const isSubstrateBalanceAvailable = async (id: string, retries = 0) => {
```

Comment:
Will need to be reworked, retry is good but currently it won't work because when you call itself recursively there is no return